### PR TITLE
Fixes reliability module issues

### PR DESCRIPTION
**Pull request by juliuspfadt**
fixes https://github.com/jasp-stats/jasp-test-release/issues/847
fixes https://github.com/jasp-stats/jasp-test-release/issues/848
fixes https://github.com/jasp-stats/jasp-test-release/issues/846
fixes https://github.com/jasp-stats/jasp-test-release/issues/887
fixes https://github.com/jasp-stats/jasp-test-release/issues/889
fixes https://github.com/jasp-stats/jasp-test-release/issues/891

### CODE REVIEW

**Comment by juliuspfadt** [comment]
In addition, the PR fixes issues with
i) the footnotes when Bayesian - missings - listwise deletion- footnotes
ii) the footnote when variables are deleted from the window, and the footnote would be empty, which crashed then
iii) being able to select no reliability coefficient, but item statistics such as item-rest correlation

File: JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R [hunk 1]
```diff
@@ -1,5 +1,7 @@
 reliabilityFrequentist <- function(jaspResults, dataset, options) {
 
+  sink("~/Downloads/log_freq.txt")
+  on.exit(sink(NULL))
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R:4]: Remove this please

File: JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R [hunk 2]
```diff
@@ -436,9 +438,24 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
       tb <- cbind(tb, newtb)
     }
 		itemTable$setData(tb)
+		
+		if (!is.null(unlist(options[["reverseScaledItems"]]))) {
+		  itemTable$addFootnote(sprintf(ngettext(length(options[["reverseScaledItems"]]),
+		                                          "The following item was reverse scaled: %s. ",
+		                                          "The following items were reverse scaled: %s. "),
+		                                 paste(options[["reverseScaledItems"]], collapse = ", ")))
+		}
```
Comment (vandenman) [inline JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R:447]: Please don't mix tabs and spaces and use 2 spaces for indents in R code.

File: JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R [hunk 3]
```diff
@@ -644,7 +661,7 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 	  g <- g + ggplot2::geom_line(data = datPrior, mapping = ggplot2::aes(x = x, y = y),
 	                              linetype = "dashed", size = .85) +
 	           ggplot2::scale_x_continuous(name = nms, breaks = xBreaks, expand = xExpand,
-	                                       limits = c(min(xBreaks), max(xBreaks)))
+	                                       limits = c(range(xBreaks)))
```
Comment (vandenman) [inline JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R:664]:
```suggestion
	                                       limits = range(xBreaks))
```
`c()` doesn't do anything right?

**Review by vandenman — CHANGES_REQUESTED** [review]
Code looks ok, I'll take a look at the behavior in JASP now.

**Comment by vandenman** [comment]
- [x] fixes jasp-stats/jasp-test-release#887
- [x] fixes jasp-stats/jasp-test-release/issues/889 
- [x] fixes jasp-stats/jasp-test-release/issues/891
- [x] fixes jasp-stats/jasp-test-release#848

- [ ] I couldn't test jasp-stats/jasp-test-release#847
image

- [ ] Doesn't fix jasp-stats/jasp-test-release/issues/846

image

open the jasp file in the issue, click refresh all and the issue persists.

**Comment by juliuspf** [comment]
> * [ ]  I couldn't test [jasp-stats/jasp-test-release#847](https://github.com/jasp-stats/jasp-test-release/issues/847)

Well, that is probably because the R-package has not been updated.

**Pull request closed without merging**